core/src/main/java/com/scalar/db/storage/dynamo/DynamoConfig.java

```diff
@@ -6,12 +6,15 @@
 import java.util.Optional;
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Immutable
 public class DynamoConfig {
+  private static final Logger logger = LoggerFactory.getLogger(DynamoConfig.class);
 
   public static final String PREFIX = DatabaseConfig.PREFIX + "dynamo.";
-  public static final String ENDPOINT_OVERRIDE = PREFIX + "endpoint-override";
+  public static final String ENDPOINT_OVERRIDE = PREFIX + "endpoint_override";
```

Comment:
Renamed `scalar.db.dynamo.endpoint-override` to `scalar.db.dynamo.endpoint_override` for consistency.